src/meta-srv/src/election/postgres.rs

```diff
@@ -462,7 +531,10 @@ impl PgElection {
         let key = key.as_bytes().to_vec();
```

Comment:
Let's change the fn parameter to `key: &[u8]` to avoid unnecessary memory allocations.